entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+exec charites "$@"
```

Comment:
Also I suggest a blank line at the end of the file

```suggestion
exec charites "$@"

```